.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Fixes # <issue>
+
+## Proposed Changes
+
+  -
+  -
+  -
```

Comment:
I think that given that we don't have any code in this repo, we don't need to have a pull request template.